src/test/kotlin/org/rust/ide/intentions/InvertIfIntentionTest.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+class InvertIfIntentionTest : RsIntentionTestBase(InvertIfIntention()) {
+    fun `test let not available`() = doUnavailableTest(""""
+        fn foo(a: Option<i32>) {
+            if let Some(x) /*caret*/= a {} else {}
+        }"""
+    )
+
+    fun `test if without condition unavailable`() = doUnavailableTest(
```

Comment:
Please format all the tests like this:
```Kotlin
fun `test`() = doAvailableTest("""
    <code>
""", """
   <code>
""")
```
or
```Kotlin
fun `test`() = doUnavailableTest("""
    <code>
""")

Reply:
Ok!